simplq/src/components/common/ClickableLogo.jsx

```diff
@@ -4,12 +4,17 @@ import React from 'react';
 import styles from '../../styles/logo.module.scss';
 
 export default (props) => {
-  let onClick = {};
-  if (props.history) {
-    onClick = () => props.history.push('/');
-  } else if (props.onClick) {
+  let onClick = () => {
+    window.location = '/';
+  };
+  if (props.onClick) {
+    // if a handler provided, use it
```

Comment:
We could use `import { useHistory } from 'react-router-dom';` and remove passing history altogether. That would be cleaner.

Reply:
I see. Let me check.

Reply:
Done. Using useHistory was neater. On using it, it goes. to the previous scroll state of the home page, which looked odd. Hence I had to make sure that we smooth scroll to top on every mount.

It seems to be working well, we can fix if issues come later.